tests/test_api.py

```diff
@@ -1101,6 +1165,74 @@ def test_get_collection_item(config, api_):
     assert 'next' not in feature['links']
 
 
+def test_get_collection_item_crs(config, api_):
+
+    # Invalid CRS query parameter
+    req = mock_request({'crs': '4326'})
+    rsp_headers, code, response = api_.get_collection_item(
+        req, 'norway_pop', '1015',
+    )
+
+    assert code == HTTPStatus.BAD_REQUEST
+
+    # Unsupported CRS
+    req = mock_request(
+        {'crs': 'http://www.opengis.net/def/crs/EPSG/0/32633'}
+    )
+    rsp_headers, code, response = api_.get_collection_item(
+        req, 'norway_pop', '1015',
+    )
+
+    assert code == HTTPStatus.BAD_REQUEST
+
+    # Supported CRSs
+    storage_crs = 'http://www.opengis.net/def/crs/EPSG/0/25833'
+    crs_4258 = 'http://www.opengis.net/def/crs/EPSG/0/4258'
+    supported_crs_list = [storage_crs, crs_4258]
```

Comment:
Suggestion: also add http://www.opengis.net/def/crs/OGC/1.3/CRS84, as that CRS is default, should always be supported (IMHO according to the Standard), even if not specified in config. Also without the `crs` parameter the Features should be returned in that CRS (default standard GeoJSON).

Reply:
Yes, the intention for 'Part 2' support is that it should not affect existing deployments, so configuring `crs/storage_crs` is not required,  but still published in Collection metadata.